src/core_plugins/tagcloud/public/tag_cloud.less

```diff
@@ -1,5 +1,5 @@
 .tagcloud-vis {
-  position: relative;
+  position: absolute;
```

Comment:
children of flex-boxes don't stretch to full height, unless the position is absolute (
http://stackoverflow.com/questions/15381172/css-flexbox-child-height-100).

This caused the tagcloud in reporting to be empty.